### PR TITLE
Add Share to Blog Posts & Changelog

### PR DESCRIPTION
**Pull request by ItzNotABug**
## What does this PR do?

This PR adds sharing option to blog posts & changelog.

## Test Plan

Manual.

<table style="border-collapse: collapse; width: 100%;">
    <tr>
        <td style="border: 1px solid #000; min-height: 225px; text-align: center; vertical-align: middle;">
            string-attr
        </td>
        <td style="border: 1px solid #000; min-height: 225px; text-align: center; vertical-align: middle;">
            int-attr
        </td>
    </tr>
    <tr>
        <td style="border: 1px solid #000; min-height: 225px; text-align: center; vertical-align: middle;">
            owr-attr
        </td>
        <td style="border: 1px solid #000; min-height: 225px; text-align: center; vertical-align: middle;">
            twr-attr
        </td>
    </tr>
</table>

## Related PRs and Issues

N/A.

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

Yes.

### CODE REVIEW

File: src/markdoc/layouts/Post.svelte [hunk 1]
```diff
@@ -152,6 +184,38 @@
...
+                                    </span>
+
+                                    <ul class="u-flex u-gap-8">
+                                        {#each socialSharingOptions as sharingOption}
+                                            <li class="share-list-item">
+                                                <Tooltip placement="bottom" disableHoverableContent={true}>
+                                                    <button
+                                                        class="web-icon-button"
+                                                        aria-label={sharingOption.label}
+                                                        on:click="{() => handleShareClick(sharingOption)}"
```
Comment (TorstenDittmann) [inline src/markdoc/layouts/Post.svelte:202]: lets use an anchor here, buttons are not that accessible 👍🏻 

you can use the same function, change it a bit and return the `href`

File: src/routes/changelog/entry/[entry]/+page.svelte [hunk 2]
```diff
@@ -52,6 +86,37 @@
...
+                                </span>
+
+                                <ul class="u-flex u-gap-8">
+                                    {#each sharingOptions as sharingOption}
+                                        <li class="share-list-item">
+                                            <Tooltip placement="bottom" disableHoverableContent={true}>
+                                                <button
+                                                    class="web-icon-button"
+                                                    aria-label={sharingOption.label}
+                                                    on:click="{() => handleShareClick(sharingOption)}"
```
Comment (TorstenDittmann) [inline src/routes/changelog/entry/[entry]/+page.svelte:103]: lets do the same suggestion I had in the blog post 👍🏻

File: src/markdoc/layouts/Post.svelte (hunk 1 above)
Comment (ItzNotABug) [reply src/markdoc/layouts/Post.svelte:202]: What about `Copy to clipboard`?

Comment (ItzNotABug) [reply src/markdoc/layouts/Post.svelte]: Added separate tags for links and copy button.

File: src/routes/changelog/entry/[entry]/+page.svelte (hunk 2 above)
Comment (ItzNotABug) [reply src/routes/changelog/entry/[entry]/+page.svelte]: Added separate tags for links and copy button.

**Comment by TorstenDittmann** [comment]
any reason the hackernews things is missing on changelog pages?

**Comment by ItzNotABug** [comment]
> any reason the hackernews things is missing on changelog pages?

Note on the designs for this mentioned not necessary to add YCombinator on Changelog.

**Review by TorstenDittmann — APPROVED** [review]

**Pull request merged**